components/DateInput.vue

```diff
@@ -105,6 +105,15 @@ export default {
       default: false,
     },
   },
+  watch: {
+    dateResponse() {
+      if (this.check === "") {
+        this.month = this.pullDateValue(this.dateResponse, 0)
+        this.day = this.pullDateValue(this.dateResponse, 1)
+        this.year = this.pullDateValue(this.dateResponse, 2)
+      }
+    },
+  },
   data() {
```

Comment:
Linter issues: 
1. The "data" property should be above the "watch" property on line 108.eslint[vue/order-in-components](https://eslint.vuejs.org/rules/order-in-components.html)
2. The "computed" property should be above the "watch" property on line 108.eslint[vue/order-in-components](https://eslint.vuejs.org/rules/order-in-components.html)

Reply:
Fixed!